proto/config/edge_node_cluster.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package = "github.com/lf-edge/eve-api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+import "evecommon/acipherinfo.proto";
+
+message EdgeNodeCluster {
+  // cluster name, in case it has multiple cluster on the same site
+  string cluster_name = 1;
+
+  // cluster UUID
+  string cluster_id = 2;
+
+  // Cluster-Interface, for example "eth1"
+  string cluster_interface = 3;
```

Comment:
I wonder if this should reference the [logicallabel of the PhysicalIO](https://github.com/lf-edge/eve-api/blob/main/proto/config/devmodel.proto#L118-L122) (like [SystemAdapter does](https://github.com/lf-edge/eve-api/blob/main/proto/config/devmodel.proto#L67-L70)), instead of referencing the network interface name (as used by Linux) directly.

Reply:
Sure, we should change the comments to use logical-label of the interface.

Reply:
I don't see any update in the name or comment to reflect this.

Reply:
we'll add at the end of comment: ", this is the logicallabel of the physicalIO"
is this right?

Reply:
Yes.